shanghai/main.py

```diff
@@ -4,39 +4,48 @@
 
 from .core import Shanghai
 from .config import Configuration
+from .logging import current_logger, LogContext
 
 
 def exception_handler(loop, context):
-    print("exception_handler context:")
-    pprint(context)
+    import io
+    import contextlib
+    f = io.StringIO()
+    with contextlib.redirect_stdout(f):
+        print("exception_handler context:")
+        pprint(context)
```

Comment:
Maybe even add the resp. file interface to current_logger, so we can just do: `print('foobar', file=current_logger)`? Well... though this doesn't say anything about the logging level I'm afraid. And something like: `print('foobar', file=current_logger.debug)` is hackish (but possible), alternativ: `print('foobar', file=current_logger.debug_stream)` (or `current_logger.fdebug`/`current_logger.sdebug` etc.).

But then again, for now I just pass `f` as kw argument.